pkg/virt-controller/services/template.go

```diff
@@ -122,7 +122,7 @@ const (
 	VirtLauncherMonitorOverhead = "25Mi"  // The `ps` RSS for virt-launcher-monitor
 	VirtLauncherOverhead        = "100Mi" // The `ps` RSS for the virt-launcher process
 	VirtlogdOverhead            = "20Mi"  // The `ps` RSS for virtlogd
-	LibvirtdOverhead            = "35Mi"  // The `ps` RSS for libvirtd
+	VirtqemudOverhead           = "35Mi"  // The `ps` RSS for virtqemud
```

Comment:
Does this still hold?

Reply:
I checked that the overhead of the `virtqemud` process was within the set range. Technically, the value could be less, but before decrementing it I would prefer to run long range tests.